src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -260,7 +260,8 @@ public async Task Put()
                     throw licenseLimit;
                 }
 
-                if (ServerStore.DatabasesLandlord.IsDatabaseLoaded(name) == false)
+                if (ServerStore.DatabasesLandlord.IsDatabaseLoaded(name) == false && 
+                    ServerStore.DatabasesLandlord.DatabaseRecordExists(name) == false)
```

Comment:
Why did you add the second check? Any update to the actual database record will not trigger this now, would it?

Reply:
It should be only triggered for newly created databases on this node.

Reply:
Why does that matter?

Reply:
if we send UpdateDatabaseOperation while DB is still creating it will enter this method and set DatabaseDisabledException.
https://github.com/ravendb/ravendb/blob/v4.1/test/SlowTests/Issues/RavenDB_10929.cs#L25